cmd/init.go

```diff
@@ -7,7 +7,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/99designs/gqlgen/codegen"
+	"github.com/monzo/gqlgen/codegen"
```

Comment:
Most vendoring tools will let you rewrite the repo without having to change the imports, for dep it you would add this to your `Gopkg.toml` of your app using gqlgen:

```toml
[[constraint]]
  branch = "master"
  name = "github.com/99designs/gqlgen"
  source = "https://github.com/monzo/gqlgen"
```